vault/provider.go

```diff
@@ -652,24 +652,26 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		}
 	}
 
-	renewable := false
-	childTokenLease, err := client.Auth().Token().Create(&api.TokenCreateRequest{
-		DisplayName:    "terraform",
-		TTL:            fmt.Sprintf("%ds", d.Get("max_lease_ttl_seconds").(int)),
-		ExplicitMaxTTL: fmt.Sprintf("%ds", d.Get("max_lease_ttl_seconds").(int)),
-		Renewable:      &renewable,
-	})
-	if err != nil {
-		return nil, fmt.Errorf("failed to create limited child token: %s", err)
-	}
+	if tokenInfo.Data["orphan"] == false {
```

Comment:
In looking at Vault's present code, I see that this field is _always_ added and accurate. However, if someone were to edit Vault's code to sometimes omit the field, or if somebody changed which method we used to look up a token above so the response now omitted it, this could always evaluate to `false`. 

Should we instead change this to:
```
isOrphan, found := tokenInfo.Data["orphan"]
if !found {
    // return an error
}
if !isOrphan {
    // the rest 
}
```
It might provide some future-proofing just in case. :-)